corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -584,6 +584,7 @@ hqDefine("cloudcare/js/formplayer/menus/views/query", function () {
 
             this.dynamicSearchEnabled = !(options.disableDynamicSearch || this.smallScreenEnabled) &&
                 (toggles.toggleEnabled('DYNAMICALLY_UPDATE_SEARCH_RESULTS') && this.options.sidebarEnabled);
+            this.searchOnClear = (options.searchOnClear && !this.smallScreenEnabled);
```

Comment:
why is this not enabled with small screens?

Reply:
Correct me if i'm wrong, but the reason dynamic search isn't enabled on small screen is because the query view and case list view aren't visible at the same time. This is the same rationale as to why I also disabled "search on clear" for small screens

Reply:
cool yep that's right sounds good